rotkehlchen/ethchain.py

```diff
@@ -53,17 +53,43 @@ def attempt_connect(self, ethrpc_port, mainnet_check=True) -> Tuple[bool, str]:
                         'the ethereum mainnet'.format(ethrpc_port)
                     )
                     return False, message
+                if self.web3.eth.syncing:  # pylint: disable=no-member
+                    current_block = self.web3.eth.syncing.currentBlock  # pylint: disable=no-member
+                    latest_block = self.web3.eth.syncing.highestBlock  # pylint: disable=no-member
+                    return self.is_synchronized(current_block, latest_block)
+                else:
+                    current_block = self.web3.eth.blockNumber  # pylint: disable=no-member
+                    latest_block = self.query_eth_highest_block()
+                    if latest_block is None:
+                        return False, 'Could not query latest block from blockcypher.'
+                    return self.is_synchronized(current_block, latest_block)
 
             self.connected = True
             return True, ''
         else:
-            logger.warn('Could not connect to a local ethereum node. Will use etherscan only')
+            logger.warning('Could not connect to a local ethereum node. Will use etherscan only')
             self.connected = False
             message = 'Failed to connect to ethereum node at port {}'.format(ethrpc_port)
 
         # If we get here we did not connnect
         return False, message
 
+    def is_synchronized(self, current_block: int, latest_block: int) -> Tuple[bool, str]:
+        """ Validate that the local ethereum node is synchronized
+            at least 99.99% before using local blockchain
```

Comment:
Can you update the comment here to specify that it's no longer a percentage but a block height different?